packages/patternfly-3/patternfly-react-extensions/package.json

```diff
@@ -65,7 +65,7 @@
   },
   "peerDependencies": {
     "prop-types": "^15.6.1",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2"
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0"
```

Comment:
Is the intention to be consistent or do we really want to downgrade / lock at 16.4.0? 

Specifying "^16.4.0" will potentially install the latest features of 16.12.0.